.gitignore

```diff
@@ -46,3 +46,6 @@ tests/backstop/tests/
 # Jest
 coverage/
 tests/e2e/screenshots
+
+# Block output
+blocks/*/dist
```

Comment:
I believe this is no longer needed, right?

Reply:
Yup, removed it 👍🏻